v2/terraform_flannel.py

```diff
@@ -328,3 +328,13 @@ def down(self):
             self.deployer.down()
         except Exception as e:
             raise e
+
+    def collectWindowsLogs(self):
+        self.opts.workspace = "/tmp/k8s-ovn-ovs"
```

Comment:
self.opts is not a usual variable. It is a object containing the CLI arguments passed to this civ2 . We shouldn't go around setting fields on this objects where a simple variable would do just fine.